src/pages/spawn/spawnHost/SpawnHostCard.tsx

```diff
@@ -61,7 +61,7 @@ const spawnHostCardFieldMaps = {
   "Home Volume": (host: MyHost) => (
     <span>
       <StyledRouterLink to={getSpawnVolumeRoute(host?.homeVolumeID)}>
-        {host?.homeVolumeID}
+        {host?.homeVolume?.uiDisplayName ?? ""}
```

Comment:
This should fall back to the `homeVolumeID` not an empty string

Reply:
this handles the case where host is `nil` or `homeVolume` is nil. `uiDisplayName` will always be a non-empty string

Reply:
I will utilize this field in other parts of spruce before I re request review